data/operators.csv

```diff
@@ -76,13 +76,15 @@ NetCologne,ISP,,unsafe,8422,873
 Bredband2,ISP,signed + filtering,safe,29518,879
 Turknet,ISP,,unsafe,12735,954
 iiNet Limited,ISP,,unsafe,4739,975
+Selectel,cloud,,unsafe,49505,1081
```

Comment:
Could we either name them differently OR keep the highest ranked?

Reply:
I guess we could use the AS name from above.